src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,169 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidAgeException;
+import core.basesyntax.exception.InvalidArgumentException;
+import core.basesyntax.exception.InvalidLoginException;
+import core.basesyntax.exception.InvalidPasswordException;
+import core.basesyntax.exception.UserAlreadyExistException;
+import core.basesyntax.model.User;
+import java.util.Random;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "ValidLogin";
+    private static final String VALID_PASSWORD = "ValidPassword";
+    private static final int VALID_AGE = 20;
+    private static final int MIN_VALID_AGE = 18;
+    private static final String FIVE_CHARS_STRING = "12345";
+    private static final String SIX_CHARS_STRING = "123456";
+    private static final String EMPTY_STRING = "";
+    private static RegistrationService registrationService;
+    private User defaultUser;
+    private Random random = new Random();
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        defaultUser = new User(VALID_LOGIN, VALID_PASSWORD, VALID_AGE);
+    }
+
+    @Test
+    void register_validValues_Ok() {
+        User actualUser = registrationService.register(defaultUser);
+        assertEquals(defaultUser, actualUser);
+    }
+
+    @Test
+    void register_nullAge_NotOk() {

```

Comment:
```suggestion
    void register_nullAge_notOk() {
```
please fix the same issues in all places